File: packages/test_echo_pkg/scripts/test_echo.py
#!/usr/bin/env python
import rospy

if __name__ == "__main__":
    rospy.init_node("PY_Echo_node")
    print("Sample output through a Python ROS Node.")
    try:
        input("Press ENTER to exit")
    except SyntaxError:
        exit(0)

File: packages/test_echo_pkg/src/test_echo.cpp
#include "ros/ros.h"
#include "iostream"
using namespace std;

int main(int argc, char **argv)
{
    // Initialize node
    ros::init(argc, argv, "CPP_Echo_node");
    ros::NodeHandle nodeHandler;
    // Echo things
    cout << "Sample output through a C++ ROS Node." << endl;
    cout << "Press ENTER to exit" << endl;
    cin.get();
    return 0;
}

